src/main/java/madstax/view/screen/CoursePlannerScreen.java

```diff
@@ -101,6 +101,7 @@ public void selectFirstRow() {
     }
 
     public int getSelectedRow() {
+        table.repaint();
```

Comment:
This looks a bit weird!

Reply:
I agree! But it was the only way I could think to stop it selecting multiple rows

Reply:
Actually ignore this - I can remove it now due to the change in how I get the selected row!